tests/drivers/fprotect/positive/prj.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=8192
+CONFIG_TEST_USERSPACE=n
+CONFIG_USERSPACE=n
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_FLOAT=y
```

Comment:
Is this intentional?

Reply:
I think it shouldn't be needed any more. Will try to remove it.

Reply:
Done

Reply:
Done